CONTRIBUTING-development.md

```diff
@@ -0,0 +1,149 @@
+# Coding style
+
+* We almost always use **CamelCase**, when naming functions, variables etc., **not snake_case**.
+
+* We also have the habits of using a **lowerCamelCase** for some variables named from their type or from their function name, as shown in the examples:
+
+  ```c
+  AlterTableCmd *alterTableCommand = NULL;
+  
+  ..
+  ..
+  
+  bool
+  IsAlterTableRenameStmt(RenameStmt *renameStmt)
+  {
+    bool isAlterTableRenameStmt = false;
+    ..
+  }
+
+* We **start functions with a comment**:
```

Comment:
I think we should add comment if there is specific functionality that needs to be expressed in comments. Good naming should not require a comment

Reply:
Yeah but this is such an important development-convention change that needs to be taken in a team-wide discussion, would you like to initiate this? :)

Reply:
I agree good naming is does not really require a comment always. But based on my experience with different codebases having a bit of useless comments in a codebase is much better than having to few comments. If we allow functions without comments, then I think people will often stop adding comments for functions that would benefit from it. Also by forcing writing a comment, people will try their best to make it non-useless and reviewers can easier add suggestions if they think it can be improved. So I think keeping this requirement for a comment would be the best option.